source/include/core_http_client.h

```diff
@@ -663,7 +735,7 @@ HTTPStatus_t HTTPClient_AddRangeHeader( HTTPRequestHeaders_t * pRequestHeaders,
  * - #HTTP_NO_RESPONSE (No data was received from the transport interface.)
  * - #HTTP_INSUFFICIENT_MEMORY (The response received could not fit into the response buffer
  * or extra headers could not be sent in the request.)
- * - #HTTP_PARSER_INTERNAL_ERROR (Internal parsing error.)
+ * - #HTTP_PARSER_INTERNAL_ERROR (Internal parsing error.)<br><br>
```

Comment:
Could these be replaced with actual newlines? These tags just look a little out of place.

Reply:
sure